contribute-to-tidb/code-style-and-quality-guide.md

```diff
@@ -38,3 +38,24 @@ func main() {
     fmt.Println(ping.Domain())
 }
 ```
+
+## Consider using values instead of pointers
+
+Many Go programmers over-use pointers and can benefit from the immutability of values.
+Go programs often return pointers to structs with a wrapper:
```

Comment:
Shall there be a displayed line break? You can see the [render version](https://github.com/zz-jason/tidb-dev-guide/blob/8852c81dfb3d257bf894ed0a3bb9debf6dcb7b91/contribute-to-tidb/code-style-and-quality-guide.md) and I'd prefer prevent line break that does not affect render.

Reply:
BTW, what do you mean by `benefit from the immutability of values`? That sounds like an advantage but previous it writes `over-use`.

Reply:
I revised the text, maybe it is clearer now?

Reply:
Yep.